packages/std/src/imports.rs

```diff
@@ -235,6 +237,38 @@ impl Api for ExternalApi {
         }
     }
 
+    fn ed25519_batch_verify(
+        &self,
+        messages: &[Vec<u8>],
+        signatures: &[Vec<u8>],
+        public_keys: &[Vec<u8>],
```

Comment:
If we don't need owned vectors in the sections encoder, we can use `&[&[u8]]` for all of those.

Do you run into ownership pain in test code then? Maybe we need something like

```
messages: &[dyn AsRef<[u8]>]
```

Reply:
Yes, some amount o pain, that I resolved "clasically" 🙂 . Will publish as soon as tests pass.